src/frontend/test_runner/tests/testdata/insert.yaml

```diff
@@ -87,3 +89,17 @@
     create table t (v1 int, v2 real);
     insert into t select 2, 3, 4.5 from t;
   binder_error: 'Bind error: INSERT has more expressions than target columns'
+- sql: |
+    /* insert with join */
+    create table t1 (a int, b int);
+    create table t2 (c int, d int);
+    create table t3 (e int, f int);
+    insert into t1 select c, e from t2 join t3 on t2.d = t3.f
+  batch_plan: |
+    BatchInsert { table: t1 }
```

Comment:
Should we insert a exchange singleton in this case? I'm not sure whether FE can execute BatchInsert.

Reply:
FE doesn't support executing BatchInsert currently. All the dml operator is executed in CN by now.